test/test.options.ts

```diff
@@ -127,6 +136,43 @@ describe('Options', () => {
     assert.strictEqual(res.config.timeout, 12345);
   });
 
+  it('should respect global options for user agent directives', async () => {
+    const google = new GoogleApis();
+    const product = 'product';
+    const version = 'version';
+    google.options({
+      userAgentDirectives: [
+        {
+          product,
+          version,
+        },
+      ],
+    });
+    const drive = google.drive({version: 'v2', auth: 'apikey2'});
+    createNockRequestHeaders(
+      '/drive/v2/files/woot?key=apikey3',
+      'user-agent',
+      'product/version google-api-nodejs-client/5.0.2 (gzip)'
```

Comment:
Where's the `5.0.2` coming from?

Reply:
I don't know... this was a part of the headers even without specifying `userAgentDirectives`

Reply:
This value is being populated based on the current version of googleapis-common, and will break when that dependency is updated.

Can we instead switch this to:

```
/product/version google-api-nodejs-client/.* (gzip)/
```

or something like that?